### PR TITLE
Add overloads for startTrasaction taking op and description.

### LINKED ISSUE #1241: Add overloads for startTrasaction taking op and description
The idea is to make it easier and less boilerplate, today we need to:

```
ITransaction transaction = Sentry.startTransaction("ANR Button");
transaction.setOperation("task");
transaction.setDescription("desc");
```

maybe:

`ITransaction transaction = Sentry.startTransaction("ANR Button", "task", "desc");`

or

`ITransaction transaction = Sentry.startTransaction("ANR Button", "task");`

the Span interface already has these overloads

cc @Tyrrrz @brustolin in case you don't have it too

### PR DESCRIPTION
**Pull request by maciejwalkowiak**
## :scroll: Description

Add overloads for startTrasaction taking op and description.

## :bulb: Motivation and Context

Fixes #1241.

## :green_heart: How did you test it?

Unit tests

## :pencil: Checklist

- [x] I reviewed the submitted code
- [x] I added tests to verify the changes
- [ ] I updated the docs if needed
- [ ] No breaking changes

### CODE REVIEW

**Review by marandaneto — APPROVED** [review]

**Comment by marandaneto** [comment]
lets wait for the discussion #1241 before merging

**Comment by marandaneto** [comment]
> lets wait for the discussion #1241 before merging

see https://github.com/getsentry/sentry-java/issues/1241#issuecomment-779233975

**Comment by marandaneto** [comment]
> No breaking changes

it's a breaking change, overloads don't take only a `name` anymore, also, we need to update the docs

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 * Enchancement: Support @SentrySpan and @SentryTransaction on classes and interfaces. (#1243)
 * Enchancement: Do not serialize empty collections and maps (#1245)
 * Ref: Simplify RestTemplate instrumentation (#1246)
+* Enchancement: Add overloads for startTrasaction taking op and description (#1244)
```
Comment (marandaneto) [inline CHANGELOG.md:11]: good to be under the new `breaking change` section

Comment (maciejwalkowiak) [reply CHANGELOG.md]: Fixed.

**Pull request merged**